### PR TITLE
Add handlers to start and stop spinning

### PR DESCRIPTION
**Pull request by nikimanoledaki**

### CODE REVIEW

File: pkg/handlers/handlers.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	fmt.Println("starting")
+	for {
+		for i := 0; i < 100000000; i++ {
+		}
+	}
+}
+
+func Unspin(http.ResponseWriter, *http.Request) {
+	fmt.Println("stopping")
+	os.Exit(200)
```
Comment (Callisto13) [inline pkg/handlers/handlers.go]: so we just quit the program here? what happens to the pod?

Comment (nikimanoledaki) [reply pkg/handlers/handlers.go]: hm true. a less destructive way would be to just log `stopping` and that would be enough to end the `for` loop!

Comment (nikimanoledaki) [reply pkg/handlers/handlers.go]: changed it!

File: pkg/handlers/handlers.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+func Spin(http.ResponseWriter, *http.Request) {
+	log.Println("starting")
+	for {
+		for i := 0; i < 100000000; i++ {
+		}
+	}
+}
+
+func Unspin(http.ResponseWriter, *http.Request) {
+	log.Println("stopping")
```
Comment (Callisto13) [inline pkg/handlers/handlers.go]: how is this stopping the for? am i missing something?

Comment (nikimanoledaki) [reply pkg/handlers/handlers.go]: welp I just realised my mental model is totally wrong (haven’t worked with this in such a long time!)

will update tomorrow!

Comment (Callisto13) [reply pkg/handlers/handlers.go]: 👍  I think you could do something cute with channels here

Comment (nikimanoledaki) [reply pkg/handlers/handlers.go]: done :D

**Review by Callisto13 — APPROVED** [review]
noice

**Pull request merged**